src/elm/Main.elm

```diff
@@ -46,6 +46,10 @@ type alias Flags =
     , isProduction : Bool
     , version : Version
     , showCookieConsentBanner : Bool
+
+    -- Minimum version of the node that will be taken
+    -- into account when computing statistics
+    , statsVersion : Version
```

Comment:
I read `statsVersion` as there being multiple versions of the stats computation method. What about `minVersionIncludedInStats`?

Reply:
Ok, I'll rename.